internal/services/datadog/datadog_monitors_resource_test.go

```diff
@@ -0,0 +1,286 @@
+package datadog_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/datadog/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type DatadogMonitorResource struct{}
+
```

Comment:
We should add another test for when `datadog_organization` isn't passed in to confirm that use case works

Reply:
Currently we don't support this functionality by terraform.